todoApp/clients/javascript/samples/typescript/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "samples",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "tsc",
+    "prebuild": "rimraf dist/",
+    "sample": "node dist/todoItemSamples.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "@notabrand/todo-non-branded": "../..",
+    "dotenv": "latest"
+  },
+  "devDependencies": {
+    "typescript": "~5.6.2",
+    "rimraf": "5.0.5"
```

Comment:
I think we need to add `@types/node` in the dev dependencies ? https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/computefleet/arm-computefleet/samples/v1/typescript/package.json#L37-L41